pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -1130,6 +1130,12 @@ func Convert_v1_Features_To_api_Features(source *v1.Features, features *api.Feat
 			State: boolToOnOff(source.Pvspinlock.Enabled, true),
 		}
 	}
+	features.PMU = &api.FeatureState{
```

Comment:
We need to move this up in the stack as `Convert_v1_Features_To_api_Features` is called conditionally.

Reply:
+1 
However, I would simply always create an empty `Features` element by default and won't make changes here for the sake of keeping all the `features` settings in one place. Could be either `Convert_v1_Features_To_api_Features` is called or even in pkg/virt-launcher/virtwrap/api/deafults.go

Reply:
something like: 
```
func (d *Defaulter) SetObjectDefaults_Domain(in *Domain) {
    d.SetDefaults_DomainSpec(&in.Spec)
    d.SetDefaults_OSType(&in.Spec.OS.Type)
    if in.Spec.SysInfo != nil {
        d.SetDefaults_SysInfo(in.Spec.SysInfo)
    }
    if in.Spec.Features == nil {
        in.Spec.Features = &Features{}
    }
```